sublime-package.json

```diff
@@ -780,6 +780,14 @@
                 "LSP": {
                   "type": "object",
                   "markdownDescription": "The dictionary of your configured language servers or overrides for existing configurations. The keys of this dictionary are free-form. They give a humany-friendly name to the server configuration. They are shown in the bottom-left corner in the status bar once attached to a view (unless you have `\"show_view_status\"` set to `false`).",
+                  "properties": {
+                    "formatters": {
```

Comment:
I guess we can assume that there will be no client config with this name.

Reply:
Feels weird to mix client configurations and other settings but I suppose this could be considered a bad initial design...

Reply:
Alternatively we could use a different top-level key under "settings", for example "lsp_formatters". But I'd prefer to keep all LSP settings together under "LSP". And even if there were a config named "formatters" everything should still work fine, because the settings from that config and the formatter scope keys stored under the same name wouldn't influence each other.